app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.errors import VaccineError
+from app.errors import NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: str) -> None:
```

Comment:
No-no , that you returning `str` - was correct , `cafe` is not `str` type . It is Cafe class type is not it ?